File: algorithm/python/BOJ9498.py
# problem : https://www.acmicpc.net/problem/9498

import sys

score = int(sys.stdin.readline().rstrip())
answer = ''

if score >= 90:
    answer = 'A'
elif score >= 80:
    answer = 'B'
elif score >= 70:
    answer = 'C'
elif score >= 60:
    answer = 'D'
else:
    answer = 'F'

print(answer)

File: algorithm/typescript/BOJ9498.ts
// https://www.acmicpc.net/problem/9498

import fs = require("fs");

const score: number = Number(fs.readFileSync("/dev/stdin").toString());
let answer: string = "";

if (score >= 90) {
  answer = "A";
} else if (score >= 80) {
  answer = "B";
} else if (score >= 70) {
  answer = "C";
} else if (score >= 60) {
  answer = "D";
} else {
  answer = "F";
}

console.log(answer);
